tests/ui/pages/catalog/catalog-page.ts

```diff
@@ -48,8 +52,32 @@ export default class CatalogPage extends BasePage {
     return new InputField(this.page, SELECTORS.searchBar);
   }
 
-  async searchByText(text: string) {
+  async searchBy(text: string) {
     await this.searchBar.fill(text);
     await this.page.locator(SELECTORS.searchBar).press('Enter');
```

Comment:
use already present searchBar getter instead of the this.page.locator(SELECTORS.searchBar)